source/user-manual/capabilities/vulnerability-detection/cpe_helper.rst

```diff
@@ -0,0 +1,557 @@
+| **set_version_if_product_matches**       | Sets the software version if it can be found in the product name through a regular expression expressed in the `version` section.                                                                                                                                 |
++------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| **set_targethw_if_product_matches**      | Sets the software architecture if it can be found in the product name through a regular expression expressed in the `target_hw` section.                                                                                                                          |
++------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| **set_version_only_if_product_matches**  | Sets the software version if it can be found in the product name through a regular expression expressed in the `version` section. If it does not match, the CPE will not be generated.                                                                            |
++------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| **set_targethw_only_if_product_matches** | Sets the software architecture if it can be found in the product name through a regular expression expressed in the `target_hw` section. If it does not match, the CPE will not be generated.                                                                     |
++------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+
+How to translate a Windows program to CPE format
+------------------------------------------------
+
+To include a Windows program in the CPE Helper dictionary, in order to be able to look for vulnerabilities for it, it is necessary to
+know the vendor, name of product and version that Syscollector extracts from the registry. After this, it is necessary to look for
```

Comment:
`that Syscollector extracts from the registry` -> `that the package inventory provides`